rmw_connextdds_common/include/rmw_connextdds/context.hpp

```diff
@@ -91,6 +92,23 @@ struct rmw_context_impl_t
   std::string qos_ctx_namespace;
   bool override_publish_mode;
 
+  enum class qos_profile_loading_policy_t
```

Comment:
I would rename this type to `endpoint_qos_override_policy_t` to match the environment variable, which I would also modify accordingly by moving "override" after "endpoint_qos", because I think it sounds more natural.

I'll add some reminders in different places to help you catch the required updates.

Reply:
Thanks for the suggested naming, it sounds better.
Applied in https://github.com/rticommunity/rmw_connextdds/pull/7/commits/7e5fbe48e853c4708acbcf2e2d36f88f1f0f8cc9